proto/ibc/tendermint/tendermint.proto

```diff
@@ -45,6 +47,23 @@ message ClientState {
       [(gogoproto.moretags) = "yaml:\"proof_specs\""];
 }
 
+// ConsensusState defines the consensus state from Tendermint.
+message ConsensusState {
+  option (gogoproto.goproto_getters) = false;
+
+  // timestamp that corresponds to the block height in which the ConsensusState
+  // was stored.
+  google.protobuf.Timestamp timestamp = 1
+      [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  // commitment root (i.e app hash)
+  ibc.commitment.MerkleRoot root = 2 [(gogoproto.nullable) = false];
+  // height at which the consensus state was stored.
+  uint64 height = 3;
+  bytes next_validators_hash = 4 [
```

Comment:
casttype to `HexBytes`

Reply:
I don't think I can do this until tendermint is updated right?

Reply:
you can use `(gogoproto.casttype) = "github.com/tendermint/tendermint/libs/bytes.HexBytes"`

Reply:
if it breaks we can leave it as it